src/components/Jobs/Form/Form.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types'
+import { useForm } from 'react-hook-form'
+import './form.css'
+
+function Form({ formData }) {
+  const { register, handleSubmit } = useForm()
+  return (
+    <form
+      onSubmit={handleSubmit(submit => formData(submit, 1))}
+      className="form-inline text-center job-form"
+      id="jobForm"
+    >
+      <div className="zipcode-container">
+        <label className="sr-only job-label" htmlFor="zipCode">
+          Zip Code
+        </label>
+        <input
+          type="string"
```

Comment:
Type string but the regex is for numbers

Reply:
changed string to number for ZipCode